src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -51,12 +51,7 @@
     "System.Diagnostics.DiagnosticSource": "4.0.0-*"
   },
   "frameworks": {
-    "net451": {
-      "frameworkAssemblies": {
-        "System.Reflection": "",
-        "System.Runtime": ""
-      }
-    },
+    "net451": { },
```

Comment:
Just curious, shouldn't we have use the TFM `netstandard1.2` (instead which targets 451 & 452) here as its a library project?. My understanding is that we are supposed to use monikers like `net451` and `netstandardapp1.5` only in projects are actually run (ex: samples, tests).